MagneticField/Interpolation/src/MagneticFieldGrid.cc

```diff
@@ -99,9 +99,9 @@ void MagneticFieldGrid::interpolateAtPoint(double X1, double X2, double X3, floa
   int index1[3] = {0,0,0};
   for (int i=0; i<3; ++i){
     if (NumberOfPoints[i] > 1){
-                                           index0[i] = max(0,index[i]);
+      index0[i] = max(0,index[i]);
       if (index0[i] > NumberOfPoints[i]-2) index0[i] = NumberOfPoints[i]-2;
-                                           index1[i] = max(1,index[i]+1);
```

Comment:
The indentation of this line leads gcc 6.0 to warn that it might have been intended to be part of the if block on the line above. Clearly it is not, but formatting changed to remove warning.